components/coins-status/index.js

```diff
@@ -82,10 +82,17 @@ const CoinsStatus = () => {
 
   return (
     <div className={styles.coinsContainer}>
-      {isError && <p>Something went wrong ...</p>}
-      {isError && isLoginMessage && (
+      {isError && <p>Something went wrong ...&nbsp;</p>}
+      {isError && isLoginError && (
         <p>
-          Please <a href="LOGIN_URL">login to view your balance.</a>
+          Please&nbsp;
```

Comment:
Can we have `&nbsp` on next line and properly spaced? Code will look clean. 😃

Reply:
Not necessarily on the next line, but at least separated by a space maybe? Looking too cluttered now